acceptance/Rakefile

```diff
@@ -70,3 +71,48 @@ task :acceptance do
   beaker += " #{opts}" if opts != ''
   sh beaker
 end
+
+namespace :ci do
+  namespace :test do
+
+    desc 'This task is used by CI to test the package pe-client-tools'
+    rototiller_task :package => [:set_test_target]do |t|
```

Comment:
This task does not trigger the `:host_config` task to execute `beaker-hostgenerator` after the target is set.
```
$ SHA=5a51d6b9d31c7ec2eb2af4ed6b196a4c205e20b0 SUITE_VERSION=1.0.4.81.g5a51d6b CLIENT_TEST_TARGET=windows2012r2-64 be rake ci:test:package

The CLI switch '--xml' will be used.
The CLI switch '--no-color' will be used.
The CLI switch '--debug' will be used.
The CLI flag '--hosts' will be used with value 'centos7-64.mdca-windows2012r2-64client.a'.
The CLI flag '--helper' will be used with value 'helper.rb'.
The CLI flag '--keyfile' will be used with value '/home/john.duarte/.ssh/id_rsa-acceptance'.
The CLI flag '--load-path' will be used with value 'lib'.
The CLI flag '--pre-suite' will be used with value 'setup/pe/pre_suite'.
The CLI flag '--tests' will be used with value 'tests/'.
The CLI flag '--preserve-hosts' will be used with value 'onfail'.
INFO: The environment variable: 'SHA' was found with value: '5a51d6b9d31c7ec2eb2af4ed6b196a4c205e20b0': The sha for pe-client-tools
INFO: The environment variable: 'SUITE_VERSION' was found with value: '1.0.4.81.g5a51d6b': The suite version used by Jenkins
INFO: The environment variable: 'pe_dist_dir' was found with value: 'http://neptune.puppetlabs.lan/2016.3/ci-ready': The location to download PE from example "http://neptune.puppetlabs.lan/20XX.X/ci-ready"

Hosts file 'centos7-64.mdca-windows2012r2-64client.a' does not exist.
Trying as beaker-hostgenerator input.
```